src/java.base/share/classes/jdk/internal/vm/TranslatedException.java

```diff
@@ -1,10 +1,12 @@
 /*
- * Copyright (c) 2018, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
If you just moved the file the original copyright year should remain - this is not new code.